src/main/kotlin/no/nav/familie/tilbake/api/BehandlingController.kt

```diff
@@ -59,12 +59,14 @@ class BehandlingController(private val behandlingService: BehandlingService,
 
     @PostMapping(path = ["{behandlingId}/steg/v1"],
                  produces = [MediaType.APPLICATION_JSON_VALUE])
+    // Rollen blir endret til BESLUTTER i Tilgangskontroll for FatteVedtak steg
```

Comment:
Du fann ein måte å gjenbruke dette endepunktet?
For eg klarer ikkje frå koden å sjå kor tilgangsjekken for Fatte vedtak-steget blir gjort?

Reply:
Ja, jeg fant en måte å gjenbruke dette endepunktet. minimumBehandlerRolle er overstyret til Beslutter når FatteVedtak steg utføres. Sjekk TilgangAdvice klass